### PR TITLE
Change directionality of node connections

### LINKED ISSUE #1: The directionality of dependency arrows is shown backwards
In a directed graph like Mikado, the graph should flow from the root node towards the leaves. So, the arrows should point to the dependencies rather than towards the goal, although the order of eventual refactorings will flow in the opposite direction. So, the directionality of dependencies should look more like the following:

Mikado Graph

Currently, the #depends_on method shows an inverted relationship, and the README provides no documented way to change the directionality.

### PR DESCRIPTION
**Pull request by snasirca**
Fixes #1 based on [The Mikado Method](https://livebook.manning.com/#!/book/the-mikado-method/chapter-3/point-946-71-71-0) text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Or install it yourself as:
 
 ### Prerequisites
 
-Ensure you have *GraphViz* installed. On macOS you can install it using Homebrew with the following command:
```
Comment (CodeGnome) [inline README.md]: This was right the first time. [macOS](https://www.apple.com/macos/high-sierra/) with a lowercase "m" is how Apple spells it.

Comment (snasirca) [reply README.md]: 👍  fixed

**Comment by CodeGnome** [comment]
This doesn't appear to be working as of d678c45. Specifically:

```
MikadoGraph::Generator.define do
  state("State A").with_prerequisites do
    state("State B").with_prerequisites do
      state("State D") 
      state("State E") 
    end 
    state("State C").with_prerequisites do
      state("State F") 
      state("State G") 
    end 
  end 
end.generate format: 'png', path: 'foo.png'
```

still results in:

example graph

and acking the code base didn't turn up any obvious keywords for specifying the leaf-to-node flow. Am I missing something non-obvious, or is this a bug?

**Comment by snasirca** [comment]
Hey @CodeGnome. I'm seeing the generated image that you pasted as correct. Maybe its a caching issue on your end? Here is what I see:

screen shot 2017-12-25 at 5 49 02 pm

Can I also see what version of the gem you have installed? This is how I checked mine through my terminal by launching `irb`:

```
2.4.0 :001 > require 'mikado_graph'
 => true
2.4.0 :002 > MikadoGraph::VERSION
 => "0.4.0"
2.4.0 :003 >
```

`0.4.0` has the fix

**Pull request merged**